client/components/ItemActionConfirmation/forms/updateRecurringEventsForm.tsx

```diff
@@ -78,7 +78,7 @@ function getRecurringPlanningToUpdate(
     const originalCoverages: IPlanningEmbeddedCoverageMap = (original.planning_ids || [])
         .map((planningId) => plannings[planningId])
         .reduce((planningItems, planningItem) => {
-            planningItems[planningItem._id] = (planningItem.coverages ?? []).reduce(
+            planningItems[planningItem?._id] = (planningItem?.coverages ?? []).reduce(
```

Comment:
This indicates that the Planning item is not stored in the Redux store. It might be good to find out why the Event's Planning items were not loaded before opening this form.